lib/fastlane/plugin/ddg_apple_automation/actions/asana_extract_task_assignee_action.rb

```diff
@@ -12,16 +11,20 @@ def self.run(params)
         task_id = params[:task_id]
         token = params[:asana_access_token]
 
-        url = Helper::DdgAppleAutomationHelper::ASANA_API_URL + "/tasks/#{task_id}?opt_fields=assignee"
-        response = HTTParty.get(url, headers: { 'Authorization' => "Bearer #{token}" })
+        client = Asana::Client.new do |c|
+          c.authentication(:access_token, token)
+        end
 
-        if response.success?
-          assignee_id = response.parsed_response.dig('data', 'assignee', 'gid')
-          Helper::GitHubActionsHelper.set_output("asana_assignee_id", assignee_id)
-          assignee_id
-        else
-          UI.user_error!("Failed to fetch task assignee: (#{response.code} #{response.message})")
+        begin
+          task = client.tasks.get_task(task_gid: task_id)
```

Comment:
We could limit this request to only return assignee too 👍 

```suggestion
          task = client.tasks.get_task(task_gid: task_id, options: { fields: "assignee" })
```